### PR TITLE
Multiline cells in read-only mode have incorrect line-height

### PR DESCRIPTION
**Pull request by johnhwhite**
- Multiline cells in read-only mode have incorrect line-height in modern theme (GitHub Issue #96)

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height)

File: src/app/public/styles/ag-grid-styles.scss [hunk 1]
```diff
@@ -32,6 +32,13 @@ ag-grid-angular:not(.sky-ag-grid-editable) {
 
   border-top: $sky-standard-border;
 
+  .ag-cell.ag-cell-wrap-text {
+    min-height: ag-param(row-height) - 2px;
+    line-height: normal;
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/styles/ag-grid-styles.scss:37]: We discussed offline about changing the line heights to `$sky-line-height-base` for default theme and then `$sky-theme-modern-font-paragraph-line-height` for modern

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-multiline-cell-height)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**